<launch>
  <arg name="input_lane_driving_trajectory" />
  <arg name="output_scenario" />
  <arg name="output_trajectory" />

  <arg name="input_parking_trajectory" default=""/>
  <arg name="input_lanelet_map" default="" />
  <arg name="input_route" default="" />
  <arg name="input_twist" default="" />
  <arg name="is_parking_completed" default="" />

  <node pkg="topic_tools" exec="relay" name="scenario_trajectory_relay" output="log" >
    <param name="use_sim_time" value="$(env AW_ROS2_USE_SIM_TIME false)"/>
    <param name="input_topic" value="$(var input_lane_driving_trajectory)" />
    <param name="output_topic" value="$(var output_trajectory)" />
    <param name="type" value="autoware_planning_msgs/msg/Trajectory" />
  </node>

  <arg name="cmd" default="ros2 topic pub $(var output_scenario) autoware_planning_msgs/msg/Scenario '{current_scenario: LaneDriving, activating_scenarios: [LaneDriving]}'" />
  <executable cmd="$(var cmd)" name="scenario_pub" output="screen" shell="true"/>
</launch>
